index.js

```diff
@@ -1,42 +1,37 @@
 #!/usr/bin/env node
+
 const Config = require('./src/config');
 const Github = require('./src/github');
 const UI = require('./src/ui');
 
-UI.printWelcome();
+const reposCommand = require('./src/commands/repos');
+const codespacesCommand = require('./src/commands/codespaces');
 
-main().then((exitCode) => {
-  process.exit(exitCode);
-});
+UI.printWelcome();
 
-async function main() {
+const main = async () => {
   try {
     if (!Config.load()) {
       const token = await UI.promptAuth();
       Config.save(token);
     }
 
-    const repositories = await UI.getRepositories();
-    if (!repositories) {
-      Config.deleteFile();
-      return await main();
-    }
-
-    let res = await UI.promptSelectRepositories(repositories);
-    if (res.repos.length === 0) {
-      UI.printNoReposSelected();
-
-      return 0;
-    }
-
-    const reposToDelete = res.repos;
-    const repoCount = reposToDelete.length;
-    res = await UI.promptConfirmDelete(repoCount);
-
-    if (res.confirmDelete === 'Yes') {
-      await UI.deleteRepositories(reposToDelete);
-    } else {
-      UI.printNoReposDeleted();
+    const command = process.argv[2];
+
+    switch (command) {
+      case 'repos':
+      case 'repo':
+        await reposCommand();
+        break;
+      case 'codespaces':
+      case 'codespace':
+        await codespacesCommand();
+        break;
+      case 'help':
+        UI.printHelp();
+        break;
+      default:
```

Comment:
This is right in order to avoid breaking changes in how the tool is executed. However, I think this could be improved. For example, if anyone executes something like this:

```sh
ghpew any-unknown-command
```

The script will execute the `reposCommand`. Maybe we could change this behavior to print the help:

```js
default:
  if (!command) await reposCommand();
  else UI.printHelp(); // unknown command
```

Reply:
Ideally, it should be tested too...

Reply:
That makes sense, I have added the check (https://github.com/adrianmg/github-pewpew/pull/37/commits/5acd3ec23cdbd0b0b5da628d36feb503f261c232)

![image](https://user-images.githubusercontent.com/63192115/182828333-2dc6fd79-c755-43a6-9f84-361adb0984f4.png)